web/src/pages/apd/proposed-budget/CombinedActivityCosts.test.js

```diff
@@ -12,14 +12,34 @@ import {
 describe('renders correctly', () => {
   it('renders correctly in view-only mode', () => {
     const component = shallow(
-      <CombinedActivityCosts data={{}} years={['1984', '1985']} isViewOnly />
+      <CombinedActivityCosts
+        apdType="HITECH"
+        data={{}}
+        years={['1984', '1985']}
+        isViewOnly
+      />
     );
     expect(component).toMatchSnapshot();
   });
 
   it('renders correctly in standard mode', () => {
```

Comment:
Should we specify this is for HITECH?

Reply:
Yea, I was trying to avoid updating the snapshot but that's lazy. I'll update it